bin/pycbc_live

```diff
@@ -328,8 +331,9 @@ class LiveEventManager(object):
 
         cmd += '--params-file {} '.format(curr_fname)
         cmd += '--approximant {} '.format(apr)
-        cmd += '--gracedb-server {} '.format(args.gracedb_server)
-        cmd += '--gracedb-search {} '.format(args.gracedb_search)
+        cmd += '--gracedb-server {} '.format(self.gracedb_server)
+        cmd += '--gracedb-search {} '.format(self.gracedb_search)
+        # FIXME should we also pass the GraceDB labels here?
```

Comment:
This is far from self-explanatory - ie why we are not passing default labels to the snr opt instance. Suggest a much more explicit reasoning, eg 'Don't pass labels to the SNR optimization job: currently labels are only required for Early Warning analysis, which does not use SNR opt'

Reply:
(That at least will enable someone to come along and work out what the answer to this question should be in future)